components/courses/spec/policies/courses/graduate/graduate_students_policy_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Courses::Season::GraduateStudentsPolicy do
+  let (:season)  { create(:season, title: 'Test Season', status: :passed) }
```

Comment:
1. You can use `described_class` which is equal to `Courses::Season::GraduateStudentsPolicy`
2. You can use [subject](https://relishapp.com/rspec/rspec-core/v/3-6/docs/subject/explicit-subject) like this:
`subject(:policy) {described_class.new(season)}`